client/src/sensemakerService.ts

```diff
@@ -26,14 +28,14 @@ export class SensemakerService {
     return this.callZome('create_assessment', assessment);
   }
 
-  async getAssessment(assessmentEh: EntryHash): Promise<HolochainRecord> {
+  async getAssessment(assessmentEh: EntryHash): Promise<Assessment> {
```

Comment:
should we revert this back to `HolochainRecord` until we change the return type of the `get_assessment` zome function to return an assessment?

Reply:
Probably. This would seem confusing if the code isn't actually returning this object.

Reply:
I'm actually confused. What is the difference between the HolochainRecord and the actual typed value? Does it matter in the client? In the playspace, I'm doing this same thing and it works just fine. On the other hand, in the zome code, I do have to do some HDK calls and pattern matching to get the correct Rust types.

Reply:
the difference is that the `Record` contains the `Action` and the `Entry`, and the `Assessment` is serialized and thus needs to be decoded like: `decode((tenScaleRangeReadOutput.entry as any).Present.entry) as any)`

Reply:
Oh, I just read the parentheses wrong.